scheduler/src/cook/config.clj

```diff
@@ -445,6 +445,7 @@
                                  pool-selection)})))
      :kubernetes (fnk [[:config {kubernetes {}}]]
                    (merge {:default-workdir "/mnt/sandbox"
+                           :pod-condition-containers-not-initialized-seconds 60
```

Comment:
How long do containers normally take to initialize?

Reply:
I posted an example timeline internally.

This number of seconds starts from when we've already seen the `ContainersNotInitialized` reason. 

Here's the relevant k8s code that adds this condition:

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/status/generate.go#L172-L187

We can make the default higher if we think this is overly sensitive, what do you think? 

Alternatively, we can see what happens in the wild and adjust this via config.edn, since this is just the default.

Reply:
Changed the default from 60 to 120